src/util/kaldi-table-inl.h

```diff
@@ -340,6 +441,13 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     // or the user called FreeCurrent().. note,
     // if when called by user we are in this state,
     // it means the user called FreeCurrent().
+    kObjectLoaded, // Called NextScpLine() and found we already  yes         yes
```

Comment:
I guess what I'm saying is that the states kLoadSucceeded and kObjectLoaded are really the same state: it means you have the object corresponding to the data_rxfilename_ in the holder but you don't have any range object.  So I'm saying you should merge the states.  I may have mis-spoken before.